frontend/package.json

```diff
@@ -305,9 +305,9 @@
     "style-loader": "^2.0.0",
     "thread-loader": "^3.0.4",
     "ts-jest": "21.x",
-    "ts-loader": "^6.2.2",
+    "ts-loader": "8.3.0",
```

Comment:
This should have a `~` or a `^`. I don't think we need to lock to a specific version.